packages/applications-service-api/src/repositories/representation.backoffice.repository.js

```diff
@@ -46,16 +46,19 @@ const getRepresentations = async (options) => {
 	};
 
 	if (options.searchTerm) {
+		const terms = options.searchTerm.split(' ');
 		where['AND'].push({
-			OR: [
-				{ represented: { firstName: { contains: options.searchTerm } } },
-				{ represented: { lastName: { contains: options.searchTerm } } },
-				{ represented: { organisationName: { contains: options.searchTerm } } },
-				{ representative: { firstName: { contains: options.searchTerm } } },
-				{ representative: { lastName: { contains: options.searchTerm } } },
-				{ representative: { organisationName: { contains: options.searchTerm } } },
-				{ representationComment: { contains: options.searchTerm } }
-			]
+			OR: terms.map((term) => ({
+				OR: [
+					{ represented: { firstName: { contains: term } } },
+					{ represented: { lastName: { contains: term } } },
+					{ represented: { organisationName: { contains: term } } },
+					{ representative: { firstName: { contains: term } } },
+					{ representative: { lastName: { contains: term } } },
+					{ representative: { organisationName: { contains: term } } },
+					{ representationComment: { contains: term } }
```

Comment:
Would it make sense to use options.searchTerm for the fields we know are single field eg representationComment and organisationName.  That way we maintain the existing accuracy/behaviour for those fields while addressing the missed data for firstname/lastname.